fabfile.py

```diff
@@ -130,6 +130,15 @@ def setup():
                 run("ln -s -T default_images images")
                 run("ln -s -T default_styles styles")
 
+    # Create a file for personal Fabric commands
+    # but only if it's not already there
+    if not files.exists(env.rbase + "fabfile_personal.py"):
+        files.upload_template(
+            join(env.lbase, "deploy", "config_templates", "fabfile_personal.py"),
+            env.rbase + "fabfile_personal.py",
+            backup=False
+        )
+
```

Comment:
Sure. Since this makes it less discoverable, I added a note in the documentation about it, which probably should have been there in the first place anyway.